src/app/modules/main/communities/models/discord_file_item.nim

```diff
@@ -0,0 +1,45 @@
+import strformat
+type
+  DiscordFileItem* = object
+    filePath*: string
```

Comment:
These we need though because we have `setData` implemented in `DiscordFileListModel` which does assignments (unless we introduce setter methods for those, but I find this a bit overengineered at this point

Reply:
Same applies to the discord categories